Storage/Storage/Model/ProductVariation+CoreDataProperties.swift

```diff
@@ -69,28 +69,28 @@ extension ProductVariation {
 extension ProductVariation {
 
     @objc(insertObject:inAttributesAtIndex:)
-    @NSManaged public func insertIntoAttributes(_ value: Attribute, at idx: Int)
+    @NSManaged public func insertIntoAttributes(_ value: GenericAttribute, at idx: Int)
 
     @objc(removeObjectFromAttributesAtIndex:)
     @NSManaged public func removeFromAttributes(at idx: Int)
 
     @objc(insertAttributes:atIndexes:)
-    @NSManaged public func insertIntoAttributes(_ values: [Attribute], at indexes: NSIndexSet)
+    @NSManaged public func insertIntoAttributes(_ values: [GenericAttribute], at indexes: NSIndexSet)
 
     @objc(removeAttributesAtIndexes:)
     @NSManaged public func removeFromAttributes(at indexes: NSIndexSet)
 
     @objc(replaceObjectInAttributesAtIndex:withObject:)
-    @NSManaged public func replaceAttributes(at idx: Int, with value: Attribute)
+    @NSManaged public func replaceAttributes(at idx: Int, with value: GenericAttribute)
 
     @objc(replaceAttributesAtIndexes:withAttributes:)
-    @NSManaged public func replaceAttributes(at indexes: NSIndexSet, with values: [Attribute])
+    @NSManaged public func replaceAttributes(at indexes: NSIndexSet, with values: [GenericAttribute])
 
     @objc(addAttributesObject:)
-    @NSManaged public func addToAttributes(_ value: Attribute)
+    @NSManaged public func addToAttributes(_ value: GenericAttribute)
 
     @objc(removeAttributesObject:)
-    @NSManaged public func removeFromAttributes(_ value: Attribute)
+    @NSManaged public func removeFromAttributes(_ value: GenericAttribute)
```

Comment:
@pmusolino I generated the code and it didn't change this part at all. 😅 It looks like this is actually correct because the relationship name is still `attributes`. 

![image](https://user-images.githubusercontent.com/198826/94753683-39612e00-034c-11eb-9889-5e68ee9f591f.png)

It's only the type that changed. 

Anyway, I added a simple test that one of the methods here still works. 62b7e2b778b95c44ecccb2485a453108c90d2a32

Reply:
Oh great! 👍